### DISCUSSION TITLE
Considering purchasing a licence - a few questions first

### DISCUSSION THREAD

**Discussion by rea1456** [aclap-dev/video-downloadhelper#963] [Help and Q&A]
### Your question

I have been using Video download helper for a long time, and up to version 8 everything worked fine. The latest changes requires a licence to do what I always have done before, but before I purchase a licence I would like to ask a few questions first.

What I would like to do is to download the original audio stream (unchanged, unconverted, HLS stream), but I am not always at liberty to install the companion app. Will basic download (without any converting or changing for format, bitrate, etc) be supported without the companion app as before, or do this now require the companion app? Can Video download helper do anything without the companion app?

I usually work from virtual computers (virtual machines), and are constantly changing them, mostly using ubuntu. If I purchase a licence, will I be able to transfer that licence from one computer (when I abandon it) to a new virtual machine, or is one licence locked to one installation?

### Addon & CoApp version and details

N/A

**Comment by jcfrog** [comment]
hi,
you can try without coapp, the extension will tell when you need it.
Your license key can be used on all your devices, virtual or not. 
There will just be a security lock every 8 installs you will unlock with a link sent to you by email, just to be sure it's you.

**Reply by GrampaWildWilly** [reply]
You can install the CoApp without buying a license.  If you're downloading audio only, the license wouldn't kick in anyway.  The license affects whether a watermark gets written into the output file.  But I don't think it's relevant to write a watermark into an audio-only output file.

As for unlocking a license, I've just been through that exercise & documented it, complete with screenshots, over here: #928.

**Reply by rea1456** [reply]
The thing is that I often work without admin rights, so I am not permitted to install the download app everywhere I am. And the licence kicks in when I try to download a HLS stream (apparently that requires a a premium subscription?). Is there a difference between licence and premium status?

**Reply by GrampaWildWilly** [reply]
No, they're synonyms.  But if you're downloading audio only, it shouldn't matter.

And premium status or a license is never absolutely required.  If you don't have a license, there are certain restrictions.  On Firefox, the only restriction is that videos will get watermarks under certain circumstances.  That's not really a restriction, just an inconvenience meant to be an incentive to get a license.  Videos will get a watermark.  Audio-only content, being non-video, can't get a watermark, even if you don't have a license.  There is a VDH function for downloading audio only.  I would not be surprised to learn that you have to have the CoApp to do that.  But I don't think you need a license for that.  Maybe.  I don't know.  I don't work for VDH.  I'm just another user.  I have a license.  And I use the CoApp.  So I don't know.  And I use only Firefox.

On Chrome, in addition to the watermark, there is a restriction that you can't do HLS downloads any more often than 2 hours apart.  Also, VDH for Chrome does not download from YouTube.  That is a legal restriction.  Google, as the owner of both the Chrome browser & YouTube, has proclaimed that no third-party Chrome extension available on the Chrome web store shall download from YouTube.  So VDH for Chrome doesn't do that.  Maybe Google has their own Chrome extension that does it.  I don't know.  Like I said, I know only Firefox.  Firefox can't be boxed in by Google.  Firefox is independent from Google.  So VDH for Firefox quite happily downloads from YouTube.  That is probably the biggest reason for the price difference between the 2 instances of VDH.

The only requirement I know of is that certain downloads require the CoApp.  That is not synonymous with having a license/premium status.  That just means the download requires the CoApp.  Now, if you're trying to download from a pay site or just a site that requires a login, the web site might be bugging you about "premium" status.  But that's the web site, not VDH or the CoApp.  In general, people who use Chrome tend to use the term "premium status."  I gather there is some little trickery required that makes you log into your Google ID before the browser will work right.  Or something like that.  Maybe that's called "premium status."  It's an environment I know nothing about so I'm guessing.

The bottom line is that if you are downloading only audio content, then a license should not be necessary.  Now, if you're downloading mainly audio-only content but not exclusively, then there might be times you'd like to have a license.  But if you're downloading only audio content, then you shouldn't need to bother with a license.  On the other hand, it is the polite thing to do to support software you like by paying for it.  But that's all on your conscience.  I'm not going to be the one to guilt you into that.
